circle.yml

```diff
@@ -28,8 +28,11 @@ dependencies:
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
       conda install -q ciocheck -c spyder-ide --no-update-deps &&
-      if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip install -q pyqt5 spyder tornado pexpect; else conda install -q spyder tornado pexpect; fi &&
-      python setup.py build_static &&
+      if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip install -q pyqt5 tornado pexpect; else conda install -q -y spyder && conda install tornado pexpect; fi &&
+      mkdir spyder-source && cd spyder-source &&
```

Comment:
Before this line please add this one

    if [ "$CIRCLE_NODE_INDEX" = "2" ]; then pip uninstall -q -y spyder; else conda remove -q -y spyder; fi &&